docs/CODE_OF_CONDUCT.md

```diff
@@ -1,21 +1,11 @@
 # Code of Conduct
```

Comment:
should this file also be mentioned in the main readme, as it's now the only file that isn't. Or actually include it in 'Contributing.md', as it's sort of a prerequisite for that?

Reply:
It is the first link in [CONTRIBUTING](https://github.com/rodekruis/IBF-system/blob/docs.update-readme/docs/CONTRIBUTING.md#contributing).